### ISSUE TITLE
Multisig address support setting up synchronized start block

### ISSUE THREAD

**Issue by Danie0918** [Magickbase/neuron-public-issues#353]
Currently in light client mode, when adding a new Multisig address, the sync will start at 0, causing the wallet sync progress to stop.

1. Optimize the display of Multisig sync data so it is not confused with wallet sync.
2. Multisig address support setting up synchronized start block.

**Comment by Danie0918** [comment]
![Image](https://github.com/Magickbase/neuron-public-issues/assets/122437870/5702c35c-dd8f-4ed8-872f-c6a9be367d6e)

**Comment by Kirl70** [comment]
Design Draft(13-1--13-3):
https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=37438%3A49658&mode=design&t=lyyUkJS0rwgDOFOY-1
<img width="882" alt="image" src="https://github.com/Magickbase/neuron-public-issues/assets/106060898/10276d3c-4ec3-4fe3-8dc3-59716b38f28c">

**Comment by yanguoyu** [comment]
@Danie0918 
> Multisig addresses need to keep the current window open for synchronization.

Does `the current window` mean the multisig address dialog?

**Comment by Danie0918** [comment]
> @Danie0918
> 
> > Multisig addresses need to keep the current window open for synchronization.
> 
> Does `the current window` mean the multisig address dialog?

Yes, here I want to make multisig address synchronization independent from wallet synchronization to avoid interfering with each other.

**Comment by yanguoyu** [comment]
> Yes, here I want to make multisig address synchronization independent from wallet synchronization to avoid interfering with each other.

That's cool, could we only sync multisig addresses and stop the wallet synchronization when the dialog opens?

**Comment by Danie0918** [comment]
> > Yes, here I want to make multisig address synchronization independent from wallet synchronization to avoid interfering with each other.
> 
> That's cool, could we only sync multisig addresses and stop the wallet synchronization when the dialog opens?

It's okay.

**Comment by Danie0918** [comment]
@yanguoyu Any update?

**Comment by yanguoyu** [comment]
> @yanguoyu Any update?

The typeorm may have a bug that broke my progress. I have resolved it. Today I will create the PR.

**Comment by yanguoyu** [comment]
https://github.com/nervosnetwork/neuron/pull/3176
